kube/services/jenkins/jenkins-deploy.yaml

```diff
@@ -88,7 +88,7 @@ spec:
             port: 8080
         resources:
           limits:
-            cpu: 0.5
+            cpu: 0.9
```

Comment:
how did you come up with this number and did not just doubled the allocation to a full CPU?